lib/components/ResourceList/ResourceList.tsx

```diff
@@ -0,0 +1,55 @@
+import { ListBase, ListItem } from '../List';
+import type { ListItemSize, ListProps, ListSpacing } from '../List';
+
+export interface ResourceListItemProps {
+  id: string;
+  ownerName: string;
+  resourceName: string;
+  ownerLogoUrl?: string;
+  ownerLogoUrlAlt?: string;
+  size?: ListItemSize;
+  controls?: React.ReactNode;
+}
+
+export interface ResourceList {
+  items: ResourceListItemProps[];
+  spacing: ListSpacing;
+}
+
+export const ResourceListItem = ({
+  ownerName,
+  resourceName,
+  ownerLogoUrlAlt,
+  ownerLogoUrl,
+  size = 'md',
+  controls,
+}: ResourceListItemProps) => {
+  return (
+    <ListItem
+      avatar={{
+        name: ownerName,
+        imageUrl: ownerLogoUrl,
+        imageUrlAlt: ownerLogoUrlAlt,
+        type: 'company',
+      }}
+      title={resourceName}
+      description={ownerName}
+      size={size}
+      controls={controls}
+    />
+  );
+};
+
+export interface ResourceListProps extends Omit<ListProps, 'items'> {
+  items: ResourceListItemProps[];
+}
+
+export const ResourceList = ({ items, theme, spacing = 'md', size = 'md' }: ResourceListProps) => {
+  return (
+    <ListBase theme={theme} spacing={spacing}>
```

Comment:
Is `ResourceListProps` a bit too broad since you are not spreading the rest props in ListBase?

Could it simply be 
```
export interface ResourceListProps {
  items: ResourceListItemProps[];
  spacing?: ListSpacing;
  size?: ListItemSize;
  theme?: ListTheme;
}
``` 
or is it something I am missing? :)

Reply:
Good point. I'll update the props and spread them instead.